update_old_prs.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env zsh
+
+github_token=''
+jira_token='mail:token' #base64 encoded
+
+for i in {330..338}
+do
+   pr_title=$(
+        curl -L \
+            -H "Accept: application/vnd.github+json" \
+            -H "Authorization: Bearer $github_token" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            "https://api.github.com/repos/eskygroup/esky-content-providers/pulls/$i" --no-progress-meter | jq --raw-output .title
+    )
+
+    regex="([A-Z]{3}-[0-9]{5}).*"
```

Comment:
is this for ETS only for now?
because we have for example https://eskygroup.atlassian.net/browse/FCACHE-1059
FCACHE-1059

Reply:
Yup, it's only for our purposes now 👍